### ISSUE TITLE
为什么计算seg loss的时候使用refined_aff_label？

### ISSUE THREAD

**Issue by awei-97** [rulixiang/wetr#15]
您好：
     在阅读代码时我注意到，代码中计算seg loss的方式好像跟论文中不太一致。如下：
        seg_loss = get_seg_loss(segs, refined_aff_label.type(torch.long), ignore_index=cfg.dataset.ignore_index)
        # reg_loss = get_energy_loss(img=inputs, logit=segs, label=refined_aff_label, img_box=img_box, loss_layer=loss_layer)
        #seg_loss = F.cross_entropy(segs, pseudo_label.type(torch.long), ignore_index=cfg.dataset.ignore_index)
   这里使用的是refined_aff_label 而不是注释中的pseudo_label，和论文中不一致，我有些疑惑，恳请您解答。

**Comment by rulixiang** [comment]
你好，`refined_aff_label` 才是经过Affinity Propagation和PAR处理后的伪标签，`pseudo_label`是初始伪标签。论文中的名称并不是代码里的变量名，请注意区分。
